src/software/simulation/er_force_simulator.cpp

```diff
@@ -333,7 +336,8 @@ void ErForceSimulator::setBlueRobotPrimitiveSet(
         {
             auto& [local_vel, angular_vel] = robot_to_vel_pair_map.at(robot_id);
             setRobotPrimitive(robot_id, primitive_set_msg, blue_primitive_executor_map,
-                              world_proto, local_vel, angular_vel);
+                              world_proto, local_vel, angular_vel,
+                              robot_orient_map.at(robot_id));
```

Comment:
for yellow robots, we did a `Angle::fromDegrees(180)`, do we need to do that here as well